apps/web/next.config.js

```diff
@@ -226,7 +227,7 @@ const nextConfig = {
   async rewrites() {
     const beforeFiles = [
       // These rewrites are other than booking pages rewrites and so that they aren't redirected to org pages ensure that they happen in beforeFiles
-      ...(process.env.ORGANIZATIONS_ENABLED
```

Comment:
Earlier we were checking the emptiness of the env variable and thus there was no way to disable it by setting it to 0 or false